### ISSUE TITLE
ptw fails silently on startup if there is a syntax error

### ISSUE THREAD

**Issue by markscottwright** [joeyespo/pytest-watch#73]
With the following unit test file:
```
# file test_syntax.py
print("hello"
```
ptw will fail silently.  
```
D:\temp\t>type test_syntax.py
print("hello"

D:\temp\t>ptw

D:\temp\t>
```

If I fix the syntax error, I can start ptw.  If I introduce a syntax error later (after starting ptw, while it is still running), I get the expected SyntaxError exception:

```
Running: py.test
============================= test session starts =============================
platform win32 -- Python 3.5.3, pytest-3.1.3, py-1.4.34, pluggy-0.4.0
rootdir: D:\temp\t, inifile:
collected 0 items / 1 errors

=================================== ERRORS ====================================
_______________________ ERROR collecting test_syntax.py _______________________
d:\tools\python35\lib\site-packages\_pytest\python.py:408: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
d:\tools\python35\lib\site-packages\py\_path\local.py:662: in pyimport
    __import__(modname)
E     File "D:\temp\t\test_syntax.py", line 2
E
E       ^
E   SyntaxError: unexpected EOF while parsing
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.20 seconds ===========================
```

**Comment by blueyed** [comment]
Related previous fix: https://github.com/joeyespo/pytest-watch/pull/45

Some code part to look at: https://github.com/joeyespo/pytest-watch/blob/91bc8a1193dc99fcd51852561f8211010fe62c58/pytest_watch/config.py#L64-L68

**Comment by blueyed** [comment]
Found this when looking into it: https://github.com/pytest-dev/pytest/issues/2753.

**Comment by grantbachman** [comment]
Has there been any movement on this? I've been bitten by this bug a few times and it's confounding when you encounter it because it isn't apparent anything is even wrong. If it helps at all, I've tested PR #90 locally and it seems to fix the issue for me.

**Comment by joeyespo** [comment]
Just released v4.2.0 which includes a fix for this. Thanks for opening the issue, @markscottwright!
